js/views/itemEditVw.js

```diff
@@ -663,5 +663,10 @@ module.exports = baseVw.extend({
     this.validateDescription();
 
     return this.$('#contractForm')[0].checkValidity();
+  },
```

Comment:
This is unnecessary. You only need to override remove if you're adding some custom functionality in there, otherwise what's in place here happens by itself.